src/wlapplication_messages.cc

```diff
@@ -19,178 +19,207 @@
 
 #include "wlapplication_messages.h"
 
+#include <cstring>
 #include <iostream>
 
 #include <boost/format.hpp>
 
 #include "base/i18n.h"
 
-using std::endl;
+constexpr int kIndent = 22;
+constexpr int kDescriptionWidth = 50;
+constexpr int kMinPadding = 2;
+static void print_help_option(const char* name, const char* description) {
+	std::cout << name;
+
+	const int name_length = strlen(name);
+	bool needs_indent = false;
+	if (name_length + kMinPadding > kIndent) {
+		needs_indent = true;
+	} else {
+		for (int i = name_length; i < kIndent; ++i) {
+			std::cout << ' ';
+		}
+	}
+
+	for (;;) {
```

Comment:
IMHO this loop and its nested loops are difficult to understand, what is going on.